packages/forms/docs/10-rich-editor.md

```diff
@@ -56,7 +56,7 @@ RichEditor::make('content')
     ->toolbarButtons([
         ['bold', 'italic', 'underline', 'strike', 'subscript', 'superscript', 'link'],
         ['h2', 'h3'],
```

Comment:
Please add the alignment tools to the docs here

Reply:
Also, opinion on whether the justify option should be documented and enabled by default?

Reply:
Ok. I think hr is pretty common though.